var/spack/repos/builtin/packages/py-horton/package.py

```diff
@@ -0,0 +1,182 @@
+# Copyright 2013-2021 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+import platform
+
+
+class PyHorton(PythonPackage):
+    """HORTON: Helpful Open-source Research TOol for N-fermion systems. Copyright (C) 2011-2016 The HORTON Development Team
+       For more information, visit HORTON's website: http://theochem.github.com/horton/latest
+    """
+
+    # FIXME: Add a proper url for your package's homepage here.
+    homepage = "https://theochem.github.io/horton"
+    url      = "https://github.com/theochem/horton/releases/download/2.1.1/horton-2.1.1.tar.gz"
+
+    maintainers = ['frobnitzem']
+
+    version('2.1.1', sha256='4b3f87920d881030ba80f097326a744de2cfee5316aa4499cc9a6501f64b5060')
+
+    depends_on('python@2.7:2.99', type=('build', 'run'))
+    # note: The py-setuptools dep should be type='build', but build_systems/python.py
+    # adds "args += ['--single-version-externally-managed', '--root=/']"
+    # This conflicts with the present setuptools version (required by python2).
+    # So, it's hacked to run-dep here.
+    depends_on('py-setuptools@:44.1.0',   type='run')
+    depends_on('py-nose@1.1.2:',        type=('build'))
+    depends_on('py-numpy@1.9.1:1.16', type=('build', 'run'))
+    depends_on('py-scipy@0.11.0:1.2.3', type=('build', 'run'))
+    depends_on('py-cython@0.24.1:', type=('build', 'run'))
+    depends_on('py-h5py@2.2.1:2.10.0', type=('build', 'run'))
+    depends_on('py-matplotlib@1.0:2.2.5', type=('build', 'run'))
+    depends_on('libxc@2.2.2:',  type=('build', 'run'))
+    depends_on('libint@2.0.3: tune=cp2k-lmax-7', type=('build', 'run'))
```

Comment:
```suggestion
    depends_on('libint@2.0.3 tune=cp2k-lmax-7')
```
Is `tune=cp2k-lmax-7` actually required or just a personal preference?

Reply:
This lmax assumption is hard-coded into horton's header files.